src/time_series_parameters.jl

```diff
@@ -71,7 +71,11 @@ function TimeSeriesParameters(
 )
     last_timestamp = initial_timestamp + resolution * len
     last_initial_time = last_timestamp - resolution * horizon
-    count = length(range(initial_timestamp, step = interval, stop = last_initial_time))
+    if interval == Dates.Second(0)
+        count = 1
+    else
+        count = length(range(initial_timestamp, step = interval, stop = last_initial_time))
```

Comment:
do we need to allocate the range to calculate this count? isn't easier to do `(last - initial)/interval`

Reply:
There was actually an off-by-one error here. Fixed.

Reply:
How about the comment on the count calculation?

Reply:
GitHub seems confused on this comment. I did remove the range call, so I'm not sure what you mean. Can you review the code again? I did just add an explanatory comment, but I'm not sure you meant that.